src/rpc/rawtransaction.h

```diff
@@ -13,6 +13,9 @@ namespace interfaces {
 class Chain;
 } // namespace interfaces
 
+/** Broadcast a transaction */
+std::string BroadcastTransaction(CTransactionRef tx, bool allowhighfees = false);
```

Comment:
I also need access to this function in #14912 from other wallet RPC methods. Not really an issue, because I can always include `src/rpc/rawtransaction.h`. But perhaps create a new `src/wallet/transaction.cpp`?

I might add `BroadcastTransaction(PartiallySignedTransaction psbt, bool allowhighfees = false)` in my own PR.

Reply:
Agreed that this should be elsewhere, I will create src/wallet/transaction.cpp for it. (I was reluctant to be too original just to move a single function, but I prefer it that way also.)

Reply:
(agree with the latter, I ended up with a different approach too, the seperate file is useful enough)